src/components/Dashboard/ThisWeek/TasksDueThisWeek/TasksDueThisWeek.test.tsx

```diff
@@ -80,7 +80,7 @@ describe('TasksDueThisWeek', () => {
             subject: '1 the quick brown fox jumps over the lazy dog',
             activityType: ActivityTypeEnum.PartnerCarePrayerRequest,
             contacts: {
-              nodes: [{ hidden: true, name: 'Smith, Roger', id: '1' }],
+              nodes: [{ name: 'Smith, Roger', id: '1' }],
```

Comment:
What error caused you to remove these and the other fields from tests on other pages? Was it that they were not set up correctly to begin with?

Reply:
Mock fields I removed like this one aren't present on the query. Now TypeScript complains about them. I'm removing them since they don't do anything anyway.